.rubocop-shared.yml

```diff
@@ -15,17 +15,17 @@ Layout/ClosingParenthesisIndentation:
 Layout/DotPosition:
   EnforcedStyle: leading
 Layout/ElseAlignment:
-  Enabled: false
+  Enabled: true
 Layout/EmptyLineAfterGuardClause:
   Enabled: false
 Layout/EndAlignment:
-  Enabled: false
+  Enabled: true
 Layout/IndentFirstArgument:
   Enabled: false
 Layout/IndentFirstHashElement:
   Enabled: false
 Layout/IndentationWidth:
-  Enabled: false
+  Enabled: true
```

Comment:
I noticed some funky violations with `EndAlignment` and `ElseAlignment `. I'm going to work with them locally for the day and gain some more info on how they are to work with. 

The `EndAlignment` is a little annoying because it doesn't seem to auto fix.

Reply:
Thank you. I guess that's the reason why they were disabled...